doc/libpmem2/pmem2_config_new.3.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: manual
 Content-Style: 'text/css'
-title: _MP(PMEM2_CONFIG_NEW, 3)
+title: _MP(PMEM2\_CONFIG\_NEW, 3)
```

Comment:
Why this change?
It is jekyll variable on gh-pages _layout; I'm just not 100% sure if it will work on pmem.io.

Reply:
I works. Checked.